api/inventory/services.proto

```diff
@@ -272,6 +362,20 @@ service Services {
       body: "*"
     };
   }
+  // AddMongoDBService adds MongoDB Service.
+  rpc AddMongoDBService(AddMongoDBServiceRequest) returns (AddMongoDBServiceResponse) {
+    option (google.api.http) = {
+      post: "/v1/inventory/Services/AddMongoDB"
+      body: "*"
+    };
+  }
+  // ChangeMongoDBService changes MongoDB Service.
+  rpc ChangeMongoDBService(ChangeMongoDBServiceRequest) returns (ChangeMongoDBServiceResponse) {
```

Comment:
Please remove it. No change APIs for now. Maybe not even in 2.0 (TBD).

Reply:
> Please remove Change* messages/rpcs.
> 
> Also, do you think we could use [MongoDB URI](https://docs.mongodb.com/manual/reference/connection-string/) for Service or Agent?

We can I think, but why we don't use this in MySQL? https://dev.mysql.com/doc/refman/8.0/en/connecting-using-uri-or-key-value-pairs.html#connecting-using-uri

Is it ok it will be different from service to service?